airflow/bin/cli.py

```diff
@@ -2109,7 +2120,12 @@ class CLIFactory(object):
             'func': next_execution,
             'help': "Get the next execution datetime of a DAG.",
             'args': ('dag_id', 'subdir')
-        }
+        },
+        {
+            'func': rotate_fernet_key,
+            'help': 'Rotate all encrypted connection credentials and variables.',
```

Comment:
I think we need a bit more info here on how to use this fuctoion. Either point them at the securing doc, or mention something like "Set ``fernet_key`` to ``new_fernet_key,old_fernet_key` before running this command".